adapters/rubicon/rubicon_test.go

```diff
@@ -268,37 +270,28 @@ func TestRubiconBasicResponse(t *testing.T) {
 	an, ctx, pbReq := CreatePrebidRequest(server, t)
 
 	bids, err := an.Call(ctx, pbReq, pbReq.Bidders[0])
-	if err != nil {
-		t.Fatalf("Should not have gotten an error: %v", err)
-	}
-	if len(bids) != 3 {
-		t.Fatalf("Received %d bids instead of 3", len(bids))
-	}
+	assert.Nil(t, err, "Should not have gotten an error: %v", err)
+	assert.Equal(t, 3, len(bids), "Received %d bids instead of 3", len(bids))
+
 	for _, bid := range bids {
 		matched := false
 		for _, tag := range rubidata.tags {
 			if bid.AdUnitCode == tag.code {
 				matched = true
-				if bid.BidderCode != "rubicon" {
-					t.Errorf("Incorrect BidderCode '%s'", bid.BidderCode)
-				}
-				if bid.Price != tag.bid {
-					t.Errorf("Incorrect bid price '%.2f' expected '%.2f'", bid.Price, tag.bid)
-				}
-				if bid.Adm != tag.content {
-					t.Errorf("Incorrect bid markup '%s' expected '%s'", bid.Adm, tag.content)
-				}
-				if !reflect.DeepEqual(bid.AdServerTargeting, tag.adServerTargeting) {
-					t.Errorf("Incorrect targeting '%+v' expected '%+v'", bid.AdServerTargeting, tag.adServerTargeting)
-				}
-				if bid.CreativeMediaType != tag.mediaType {
-					t.Errorf("Incorrect media type '%s' expected '%s'", bid.CreativeMediaType, tag.mediaType)
-				}
+
+				assert.Equal(t, "rubicon", bid.BidderCode, "Incorrect BidderCode '%s'", bid.BidderCode)
+
+				assert.Equal(t, tag.bid, bid.Price, "Incorrect bid price '%.2f' expected '%.2f'", bid.Price, tag.bid)
+
+				assert.Equal(t, tag.content, bid.Adm, "Incorrect bid markup '%s' expected '%s'", bid.Adm, tag.content)
+
+				assert.True(t, reflect.DeepEqual(bid.AdServerTargeting, tag.adServerTargeting),
```

Comment:
I believe, the `Equal` method will essentially do the same for you here. Ref: https://godoc.org/github.com/stretchr/testify/assert#Equal